packages/css/src/components/breadcrumb/breadcrumb.scss

```diff
@@ -25,27 +25,22 @@
 .amsterdam-breadcrumb__list {
   break-after: avoid;
   break-inside: avoid;
-  display: flex;
-  flex-wrap: wrap;
-  gap: 0.5rem;
-  list-style: none;
 
   @include reset;
 }
 
 .amsterdam-breadcrumb__item {
-  align-items: center;
-  display: flex;
+  display: inline;
 }
 
 .amsterdam-breadcrumb__item:not(:last-child)::after {
-  background-color: var(--amsterdam-breadcrumb-separator-background-color);
-  clip-path: path("M 4.725,16 3.275,14.545 9.815,8 3.275,1.455 4.725,0 l 8,8 z");
+  background-image: var(--amsterdam-breadcrumb-separator-background-image);
+  background-repeat: no-repeat;
   content: "";
   display: inline-block;
-  height: 1rem;
-  margin-inline-start: 1rem;
-  width: 1rem;
+  height: 1ex;
```

Comment:
This results in a much smaller caret then the design. And why not use an icon here that matches the font level of the item?

Reply:
This makes the chevron icon equally large as in Figma, unless I’m missing something?

Inge and I agreed it must be as tall as the x-height of the font. That’s exactly what the `ex` unit does, with the added bonus that the icon sizes with its text.

Not sure what you mean with ‘an icon that matches the font level of the item’. If you mean inserting an icon component, that would make it more difficult to implement in plain HTML.

Reply:
Ok, I must have been comparing to the wrong chevron in Figma, I can see the "OLD breadcrumb" now has a scaled down icon.